src/button/ux-button.ts

```diff
@@ -29,12 +29,24 @@ export class UxButton implements Themable {
     if (this.theme) {
       this.styleEngine.applyTheme(this, this.theme);
     }
+
+    if (this.disabled || this.disabled === '') {

```

Comment:
From what's I've found. It seems that disabled="true" will result to false.  This stackoverflow answer references the HTML 5 spec:

https://stackoverflow.com/questions/6961526/correct-value-for-disabled-attribute

I'm almost thinking now that the original intention of that snip of code was to follow the HTML 5 spec.  I took my original code from the ux-input.ts source to stay consistent within the project.

@EisenbergEffect @ZHollingshead, does the above sound right when ux-input was written out?

After we finalize this behavior and coding for the ux-button, i'll make another pull request to move these changes into other components that use the disabled and readonly attributes.